.changeset/lazy-pumpkins-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": patch
+---
+
+changed the files name
```

Comment:
Hello Yahmooo :vulcan_salute: 

Thanks for your PR !
The comment on this changeset is really not explicit.
Can you explain that it is a crypto icon that you changed, and which one?

Reply:
Changed it!